packages/reactotron-app/App/Stores/SessionStore.js

```diff
@@ -144,6 +144,8 @@ class Session {
   handleCommand = command => {
     if (command.type === "clear") {
       this.commandsManager.all.clear()
+    } else if (command.type === "clipboard.value") {
+      console.log('clipboard', command.payload.value)
```

Comment:
We just want to console log the value? Not put it on the users computer clipboard?

Reply:
ugh... no that's a mistake.  thx